echo-api/echo-api.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+  api "com.netflix.spinnaker.kork:kork-plugins-api"
+  implementation project(':echo-model')
```

Comment:
I think it would be worth moving Event out of echo-model and into echo-api to support this, or defining an equivalent Event class in API and have the extension processor handle mapping the API event to the model event.

As it stands the plugin implementor is going to end up with a dependency on echo-model as well which I think we would want to try to avoid.

Reply:
Yeah, I wasn't sure what to do here.  I was going with the idea that, at the end of the day, the plugin implementor can just use whatever dependencies they want but I do agree - moving things into `-api` makes more sense.